app/renderer/components/Inspector/SessionInfo.js

```diff
@@ -38,7 +64,9 @@ class SessionInfo extends Component {
     const secDiff = (minDiff - minutes) * 60;
     const seconds = Math.floor(secDiff);
 
-    return `${minutes}:${(seconds < 10) ? '0' : ''}${seconds}`;
+    const checkTime = (time) => `${(time < 10) ? '0' : ''}${time}`;
```

Comment:
can we use https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/padStart instead?